documentation/asciidoc/topics/proc_applying_custom_configuration.adoc

```diff
@@ -19,22 +19,20 @@ Changing this configuration can result in error and result in service downtime f
 Use the {brandname} Helm chart to deploy clusters of fully configurable {brandname} Server instances on {openshiftshort}.
 ====
 
-
 .Prerequisites
 
 * Have valid {brandname} configuration in XML, YAML, or JSON format.
 
 .Procedure
 
-. Add your {brandname} configuration to the `data` field of a `ConfigMap` as `infinispan-config.[xml|yaml|json]`, as in the following example:
+. Add your {brandname} configuration to the `data` field of a `ConfigMap` as `infinispan-config.[xml|yaml|json]`.
+. Optionally include `log4j.xml` logging configuration.
```

Comment:
Should we omit this here and just have it in the logging section? It's valid to provide either log4j, server configuration or both.